web_src/js/features/clipboard.js

```diff
@@ -2,7 +2,15 @@ import {showTemporaryTooltip} from '../modules/tippy.js';
 
 const {copy_success, copy_error} = window.config.i18n;
 
-export async function copyToClipboard(text) {
+export async function copyToClipboard(text, shouldFetchForText) {
+  if (shouldFetchForText) {
+    try {
+      const response = await fetch(text, {method: 'GET', redirect: 'follow'});
+      text = await response.text();
+    } catch {
+      console.error(`failed to fetch text from ${text}`);
+    }
+  }
```

Comment:
Don't alter this function, at least for copying text only. If you really have to fetch, fetch outside the function and pass in the text.

Reply:
Agree. That's why I put a Request Change. https://github.com/go-gitea/gitea/pull/21629#issuecomment-1296079707

Reply:
Dropped modifications of `clipboard.js`